recipes/dhcp/dhcp4.inc

```diff
@@ -53,41 +53,49 @@ do_install_extra_files () {
 		${D}${sysconfdir}/default/dhcp-relay
 	install -m 0755 ${SRCDIR}/init-server \
 		${D}${sysconfdir}/init.d/dhcp-server
-	install -m 0644 ${SRCDIR}/default-server \
+        install -m 0644 ${SRCDIR}/default-server \
 		${D}${sysconfdir}/default/dhcp-server
 
-	rm -f ${D}${sysconfdir}/dhclient.conf
-	rm -f ${D}${sysconfdir}/dhcpd.conf
-	install -m 0644 ${SRCDIR}/dhclient.conf \
-		${D}${sysconfdir}/dhcp/dhclient.conf
-	install -m 0644 ${SRCDIR}/dhcpd.conf \
-		${D}${sysconfdir}/dhcp/dhcpd.conf
+        mv ${D}${sysconfdir}/dhcpd.conf.example \
+    	   	${D}${sysconfdir}/dhcp/dhcpd.conf.example
```

Comment:
seems like there are so tab/spaces issues here, make sure it is aligned with the current indention